### PR TITLE
modify sidebar of development-branch page 

### PR DESCRIPTION
**Pull request by bhaskarhc**
Screenshot 2020-03-31 at 4 57 06 PM

### CODE REVIEW

File: src/app/table/table.component.ts [hunk 1]
```diff
@@ -362,33 +363,86 @@ export class TableComponent implements OnInit {
...
+      };
+      obj.push(stageObj);
+    })
+    return obj
+  }
+
+  stageStatusForObj(stage, jobs) {
+    let stageJobs = jobs.filter(job => job.stage == stage);
+    // console.log('forStageObj',stageJobs);
+    let successJ = stageJobs.filter(j => j.status == "success").length
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]:
```suggestion
    let successJobs = stageJobs.filter(j => j.status == "success").length
```

File: src/app/table/table.component.ts [hunk 2]
```diff
@@ -362,33 +363,86 @@ export class TableComponent implements OnInit {
...
-    let gitlabStages = ["CLUSTER-SETUP", "PROVIDER-INFRA-SETUP", "STATEFUL-APP-DEPLOY", "APP-FUNC-TEST", "APP-CHAOS-TEST", "APP-DEPROVISION", "CLUSTER-CLEANUP"];
-    this.clusterSetupStatus = this.getStageStatus(gitlabStages[0], pipelineData[index].jobs)
-    this.providerInfraSetup = this.getStageStatus(gitlabStages[1], pipelineData[index].jobs)
-    this.statefulAppDeploy = this.getStageStatus(gitlabStages[2], pipelineData[index].jobs)
-    this.appFunctionTest = this.getStageStatus(gitlabStages[3], pipelineData[index].jobs)
-    this.appChaosTest = this.getStageStatus(gitlabStages[4], pipelineData[index].jobs)
-    this.appDeprovision = this.getStageStatus(gitlabStages[5], pipelineData[index].jobs)
-    this.clusterCleanup = this.getStageStatus(gitlabStages[6], pipelineData[index].jobs)
-    this.barChart(pipelineData[index]);
+    // this.barChart(pipelineData[index]); // disabled for BarChart Disabled
```
Comment (chandankumar4) [inline src/app/table/table.component.ts]: Is this required?

File: src/app/table/table.component.ts (hunk 1 above)
Comment (bhaskarhc) [reply src/app/table/table.component.ts]: 👍

File: src/app/table/table.component.ts (hunk 2 above)
Comment (bhaskarhc) [reply src/app/table/table.component.ts:445]: yes .. Required .

**Pull request merged**